### ISSUE TITLE
Add Windows Upgrade instructions

### ISSUE THREAD

**Issue by pwonson** [rundeck/docs#728]
https://docs.rundeck.com/docs/upgrading/upgrading.html#general-upgrade-guide

Enterprise NOTE ..... Before performing the upgrade, we recommend going the cluster manager of your instance that's going to be upgraded and move the scheduled jobs for this instance to a different instance.

1.Download the latest Rundeck version .war

https://download.rundeck.com/3.3.6/index.html

2.copy or move the downloaded rundeck.war file into your $Rdeck_Base folder

3.Edit the start_rundeck.bat file inside your $Rdeck_Base dir and change the name of your rundeck.war file with the name of the downloaded file "rundeckpro-enterprise-3.3.6-20201111.war"

E.g.: java %RDECK_CLI_OPTS% %RDECK_SSL_OPTS% -jar rundeckpro-enterprise-3.3.6-20201111.war --skipinstall -d >> %CURDIR%\var\logs\service.log 2>&1

4.Backup your $Rdeck_Base\libext folder (in case you have any custom plugins)

  Backup your configuration files from $Rdeck_Base/server/config dir like login modules, SSL properties, rundeck-config.properties, framework.properties files, ACL policies etc. (just in case)

5.Stop Rundeck Service

6.Delete these files and dirs: 

-$RDECK_BASE/libext/ $RDECK_BASE/tools/ $RDECK_BASE/server/lib/ $RDECK_BASE/server/sbin/ $RDECK_BASE/var/.firstLogin $RDECK_BASE/var/.first-run-3.0.22-20190512 $RDECK_BASE/var/.install-complete-missing-ver

8.Run the Rundeck service in installonly mode
java -jar rundeck-3.3.6-20201111.war --installonly

9.Restore your custom backed up custom plugins (not the whole libext folder) and configuration file from /etc/rundeck dir

10.Start your Rundeck Service

11. set execution mode to "active".

12. takeover scheduled jobs:
Go to the Rundeck UI, top-right gear icon, Cluster Manager, Jobs tab, Select All checkbox, click Action button, Change Schedule Owner option, select server member and click Change schedule owner button.

**Comment by sebastianbello** [comment]
* UPDATED Procedure for Windows

**Before performing this upgrade, we highly recommend you stop all cluster members and do the following steps, one cluster member, at a time.**

1) Download the latest Rundeck version .war
eg. 
https://download.rundeck.com/3.3.6/index.html

2) Copy or move the downloaded rundeck war file into your` $RDECK_BASE` folder

3) Stop Rundeck (stop service or kill the process)

4) Edit the file start_rundeck.bat located on your `$RDECK_BASE` dir and change the name of your rundeck.war file with the name of the downloaded file, e.g. "rundeckpro-enterprise-3.3.6-20201111.war"

```
 java %RDECK_CLI_OPTS% %RDECK_SSL_OPTS% -jar rundeckpro-enterprise-3.3.6-20201111.war --skipinstall -d >> %CURDIR%\var\logs\service.log 2>&1

```

5) Backup your `$RDECK_BASE\libext` folder (in case you have any custom plugins)

6) Backup both folders, `$RDECK_BASE/server/config` and` $RDECK_BASE/etc`, just in case

7) Delete these files and dirs: 

```
$RDECK_BASE/libext/
$RDECK_BASE/tools/
$RDECK_BASE/server/lib/
$RDECK_BASE/server/sbin/
$RDECK_BASE/var/.firstLogin
$RDECK_BASE/var/.first-run-3.0.22-20190512
$RDECK_BASE/var/.install-complete-missing-ver
```

8) Start Rundeck from the console and wait until it boots up successfully

```
java -jar rundeckpro-enterprise-3.3.6-20201111.war

```

ref: 
General Upgrade Procedure for War upgrade in Cluster
https://github.com/rundeck/docs/issues/733

**Comment by fdevans** [comment]
Closed with #734
